addon/components/bs-button.hbs

```diff
@@ -1,9 +1,10 @@
 <button
   disabled={{this.__disabled}}
   type={{this.buttonType}}
```

Comment:
This could be also changed to avoid a local variable:

```suggestion
  type={{if @buttonType @buttonType "button"}}
```

It's a little bit more verbose but doesn't require a switch to another file in order to get it. It's an alternative pattern for default values.

Reply:
Yes, that would work. I thought about `{{or @buttonType "button"}}` but `or` is not available be default yet.

Reply:
Yeah. That's even shorter syntax for default values in template. I like that one.

Reply:
Yes, but wo don't have this helper available yet. Would need `ember-truth-helpers`, until we have RFC562. But then I would rather add a simple `{{bs-or}}` or better maybe `{{bs-default}}` (I don't really like abusing boolean logic for defaults :wink::)

Reply:
I did some more work on refactoring `<BsAlert>`, and there you have both the template and the class read some args with a default value required. In that case `@arg` seems much better, as you have a single source of truth. Same happens when using the same arg multiple times in a template, as you can easily introduce errors by not having the (same) default value used everywhere...

Related: https://github.com/emberjs/rfcs/issues/479